tests/e2e/support/commands.js

```diff
@@ -164,19 +153,17 @@ Cypress.Commands.add('signupOpponent', (username, password) => {
     );
   });
 });
-
-Cypress.Commands.add('signupPlayer', (username, password) => {
+Cypress.Commands.add('signupPlayer', (player) => {
   cy.window()
     .its('cuttle.app.config.globalProperties.$store')
-    .invoke('dispatch', 'requestSignup', { username, password });
-  cy.log(`Signed up player ${username}`);
+    .invoke('dispatch', 'requestSignup', { username: player.username, password: player.password });
+  cy.log(`Signed up player ${player.username}`);
 });
-
-Cypress.Commands.add('loginPlayer', (username, password) => {
+Cypress.Commands.add('loginPlayer', (player) => {
   cy.window()
     .its('cuttle.app.config.globalProperties.$store')
-    .invoke('dispatch', 'requestLogin', { username, password });
-  cy.log(`Logged in as player ${username}`);
+    .invoke('dispatch', 'requestLogin', { username: player.username, password: player.password });
+  cy.log(`Logged in as player ${player.username}`);
```

Comment:
In which case we'd do the same move with the `.dispatch()` call arguments since `username` and `password` would already be defined